parts/k8s/cloud-init/artifacts/cse_customcloud.sh

```diff
@@ -80,14 +80,15 @@ configureK8sCustomCloud() {
     # shellcheck disable=SC2002,SC2005
     echo $(cat "${AZURE_JSON_PATH}" | jq '.tenantId = "adfs"') >${AZURE_JSON_PATH}
   fi
+  set -x
 
-  # Decrease eth0 MTU to mitigate Azure Stack's NRP issue
-  echo "iface eth0 inet dhcp" | sudo tee -a /etc/network/interfaces
-  echo "    post-up /sbin/ifconfig eth0 mtu 1350" | sudo tee -a /etc/network/interfaces
-
-  ifconfig eth0 mtu 1350
+    {{- if not IsAzureCNI}}
```

Comment:
nit: unnecessary indentation

(the pattern we've been following thus far is not to indent actual static chars [e.g., the shell script in this case], so you can put the `{{` and the echo statements on a common indentation indentation alignment at the beginning of the line)